packages/pigeon/lib/java_generator.dart

```diff
@@ -145,6 +145,17 @@ String _javaTypeForDartType(String datatype) {
   return _javaTypeForDartTypeMap[datatype];
 }
 
+String _mapGetter(Field field, List<Class> classes, String mapName) {
+  final HostDatatype hostDatatype =
+      getHostDatatype(field, classes, _javaTypeForDartType);
+  final String result = '$mapName.get("${field.name}")';
+  if (field.dataType == 'int') {
```

Comment:
test?

Reply:
I'll look into it.  I have e2e tests for iOS, not Android, but they aren't checked in yet.

Reply:
I'm going to check this in now since I can repro this manually with the video player plugin.  I'll put e2e android on my queue.